src/chocolatey/infrastructure.app/rules/LicenseMetadataRule.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright © 2023-Present Chocolatey Software, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+//
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("chocolatey.tests, PublicKeyToken=uetoasueoa")]

```

Comment:
We were going to come back to talk about this later, right?

Reply:
Yeah, it wasn't meant to be included. I'll get it removed.